packages/react-components/src/trace-explorer/trace-explorer-opened-traces-widget.tsx

```diff
@@ -111,8 +109,9 @@ export class ReactOpenTracesWidget extends React.Component<ReactOpenTracesWidget
         event.stopPropagation();
     }
 
-    protected doHandleOpenedTracesUpdatedSignal(): void {
-        if (!this.props.widgetManager.getWidgets(this.props.traceViewerWidgetID).length){
+    protected doHandleOpenedTracesUpdatedSignal(payload: OpenedTracesUpdatedSignalPayload): void {
+        this._numberOfOpenedExperiments = payload.getNumberOfOpenedTraces();
```

Comment:
the value in the payload is the opened traces in the trace explorer and server in that respect, and not the number of open tabs. So, I need to find a different way to notify about opened tabs.